dev-python/pytest-rerunfailures/pytest-rerunfailures-9.1.1.ebuild

```diff
@@ -1,9 +1,9 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
 
-PYTHON_COMPAT=( python3_{7..9} pypy3 )
+PYTHON_COMPAT=( python3_{7..10} pypy3 )
```

Comment:
This one fails tests on all impls for me. I'm going to skip this commit for now and try to fix it first.